### PR TITLE
Update Swagger Codegen from 2.1.6 to 2.2.2

### PR DESCRIPTION
**Pull request by yugui**

### CODE REVIEW

**Comment by yugui** [comment]
CLA bot is failing ...

File: examples/clients/abe/examplepb_numeric_enum.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ *
+ * OpenAPI spec version: version not set
+ * 
+ * Generated by: https://github.com/swagger-api/swagger-codegen.git
+ */
+
+package abe
+
+// NumericEnum is one or zero.   - ZERO: ZERO means 0  - ONE: ONE means 1
+type ExamplepbNumericEnum struct {
```
Comment (achew22) [inline examples/clients/abe/examplepb_numeric_enum.go]: Do you know why the enum values are empty?

Comment (yugui) [inline examples/clients/abe/examplepb_numeric_enum.go]: I don't know why. But this PR is neutral to the issue.
https://github.com/grpc-ecosystem/grpc-gateway/pull/415/files#diff-743714d76d1753794032b45e364fd780

Can we work on the issue in another PR?

Comment (achew22) [inline examples/clients/abe/examplepb_numeric_enum.go]: 100%. Would you be willing to file a bug?

**Pull request merged**